[WASH-981] Update current limitations for plugin

Mrhea opened the pull request.

Commit:
[WASH-981] Update current limitations for plugin

- Supports both Android and iOS
- Works with mixed case realm names
- Plugin compatible with TozID identities created from other
  SDKs and via TozID UI

rosasck commented on README.md:
```diff
@@ -101,7 +101,4 @@ I/flutter ( 6719): Wrote file Record {record_id: 0901582a-1ae2-4e42-8435-f74212e
-* Plugin implements a partial set of all the methods and classes provided by the native SDK ([e3db-java](https://github.com/tozny/e3db-java)).
-* Plugin only works with TozId Realms where the realm name is all lowercase.
-* Plugin only works with TozId Identities creating using this plugin / is incompatible with those created via the TozId UI.
+* Plugin implements a partial set of all the methods and classes provided by the native SDK ([e3db-java](https://github.com/tozny/e3db-java)/[e3db-swift](https://github.com/tozny/e3db-swift)).
```
Should we list the functionality? or is there examples to show

galxy25 replied:
Could reference the example app or the flutter code example in this readme

Mrhea replied:
Added relative link to example app and reference to the flutter code example

Commit:
[WASH-981] Add README ref for supported methods

- Adds references to README for where the supported SDK classes
  and methods may be found
